app/javascript/bundles/HelloWorld/components/HelloWorld.jsx

```diff
@@ -1,20 +1,37 @@
-import PropTypes from 'prop-types';
 import React from 'react';
+import PropTypes from 'prop-types';
+import LoginPage from './LoginPage'
+import MainMenu from './MainMenu'
+import Settings from './Settings'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
+import { Container } from 'semantic-ui-react'
+import { Button } from 'semantic-ui-react'
+import '../styles/office-killers'
 
 export default class HelloWorld extends React.Component {
-  static propTypes = {
-    name: PropTypes.string.isRequired, // this is passed from the Rails view
-  };
-
-  /**
-   * @param props - Comes from your rails view.
-   */
   constructor(props) {
     super(props);
 
-    // How to set initial state in ES6 class syntax
-    // https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class
-    this.state = { name: this.props.name };
+    this.state = {
+      loggedInStatus: false,
+      user: {},
+      current_userId: this.props.id,
+      first_name: this.props.first_name,
+      last_name: this.props.last_name,
+      imageURL: this.props.imageURL
+     };
+  }
+
+  componentDidMount() {
+    this.checkLogined();
+  }
+
+  checkLogined = () => {
+    if (this.state.current_userId == null) {
+      this.setState({loggedInStatus: false});
+    } else {
+      this.setState({loggedInStatus: true});
```

Comment:
`this.setState({loggedInStatus: !!this.state.current_userId});`

Reply:
checkLogined move to initial state:

this.state = {
....
loggedInStatus: !!props.id
}